python/mhs/import_scenarios.py

```diff
@@ -249,12 +274,13 @@ def import_event_set(es):
     verbose_message("Model contains {0} events\n" .format(len(es.events)))
 
     with connections['hazard_contrib'].cursor() as cursor:
+        License.load_licenses(cursor)
         # TODO investigate commit/rollback
         event_set_id = _import_event_set(cursor, es)
-        # TODO _import_contribution(cursor, ex, event_set_id)
+        _import_contribution(cursor, event_set_id, es.contribution)
         verbose_message('Inserted event_set, id={0}\n'.format(event_set_id))
         _import_events(cursor, event_set_id, es.events)
-        verbose_message('Updateding bounding box\n')
+        verbose_message('Updateing bounding box\n')
```

Comment:
Is `Updateing` a valid spelling of `Updating`?

Reply:
No.  Fix coming soon...